### PR TITLE
Began front-end

### PR DESCRIPTION
**Pull request by bhambleton123**
Edited database and seed files, got static portion of bookings module almost complete.

### CODE REVIEW

File: client/components/calendar/calendar.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    for(let i=1; i<=7-startOfMonthDay; i++){
+      week.push(i);
+    }
+
+    month.push(week);
+
+    let count = 0;
+    week = [];
+
+    for(let i=8-startOfMonthDay; i<=daysInMonth; i++){
```
Comment (RachNovo) [inline client/components/calendar/calendar.jsx]: You could fix the spacing in your for loops.

**Comment by RachNovo** [comment]
Lgtm! :) Except some spacing issues (see comment)

File: client/components/calendar/calendar.jsx (hunk 1 above)
Comment (bhambleton123) [reply client/components/calendar/calendar.jsx]: Thanks for the tip!

File: client/components/calendar/calendar.jsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+class Calendar extends Component {
+  constructor(props){
+    super(props);
+
+    this.state = {
+      date: moment(),
+      weeks: [],
+      month: '',
+      year: moment().month(moment().month()).format('YYYY'),
```
Comment (therobinkim) [inline client/components/calendar/calendar.jsx]: Does `moment` really require you to write `moment().month(moment().month())`?

File: client/components/calendar/calendar.jsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    this.dateLeft = this.dateLeft.bind(this);
+    this.renderWeeks = this.renderWeeks.bind(this);
+  }
+
+  componentDidMount(){
+    this.renderWeeks(moment().month());
+  }
+
+  async dateRight(){
+    await this.setState({
```
Comment (therobinkim) [inline client/components/calendar/calendar.jsx]: Does `this.setState` offer a Promise API that you can `await`..?

File: client/components/calendar/calendar.jsx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+      incrementMonth: this.state.incrementMonth + 1
```
Comment (therobinkim) [inline client/components/calendar/calendar.jsx]: Use the callback form of `this.setState`: https://reactjs.org/docs/react-component.html#setstate

File: client/components/calendar/calendar.jsx [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+  async dateLeft(){
+    await this.setState({
+      incrementMonth: this.state.incrementMonth - 1
+    })
+
+    await this.setState({
+      year: moment().month(moment().month() + this.state.incrementMonth).format('YYYY')
+    })
+
+    this.renderWeeks(moment().month() + this.state.incrementMonth);
```
Comment (therobinkim) [inline client/components/calendar/calendar.jsx]: There is no guarantee that `this.state.incrementMonth` has been updated yet.

File: client/components/calendar/calendar.jsx [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+    //push remaining weeks into month array
+    this.setState({
+      weeks: month,
+      month: moment().month(curMonth).format('MMMM'),
+    })
+  }
+
+  render(){
+    let weeksArray = ['sun', 'mon', 'tue', 'wed', 'thurs', 'fri', 'sat'];
```
Comment (therobinkim) [inline client/components/calendar/calendar.jsx]: Should `thurs` have 3 letters, like the other days of the week in this array?

**Pull request merged**